docs/villagedefense/setup/free-maps.md

```diff
@@ -38,3 +36,10 @@ Link to all maps of our minigames [at GDrive](https://drive.google.com/open?id=1
 
 ![](https://i.imgur.com/GMmTz1b.jpg)
 
+### Mushroom <a id="mushroom"></a>
+
+**Download link:** [Mega](https://mega.nz/folder/Zo5jWYyQ#eva37vtsdIscuTw3NTCXqA)
```

Comment:
Change download links to:

Quarry: https://download.plugily.xyz/get.php?map=Quarry&type=vd
Hollow: https://download.plugily.xyz/get.php?map=Hollow&type=vd
Mushroom: https://download.plugily.xyz/get.php?map=Mushroom&type=vd